src/app/models/AudioRecording.ts

```diff
@@ -94,21 +90,6 @@ export class AudioRecording
     return this.playUrl;
   }
 
-  public get detailsUrl(): string {
-    return this.getDetailsUrl();
-  }
-
-  public getDetailsUrl(
-    project?: IdOr<Project>,
-    region?: IdOr<Region>,
-    site?: IdOr<Site>
-  ): string {
-    return audioRecordingMenuItem.route.format(
-      { audioRecordingId: this.id },
-      { projectId: id(project), regionId: id(region), siteId: id(site) }
-    );
-  }
-
```

Comment:
seems inconsistent with other models to remove this.

Without context, I think it would be fine to route this to the base route.